backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/callout/C_Order.java

```diff
@@ -179,9 +179,10 @@ public void setContactDetails(final I_C_Order order)
 
 		if (userRecord.isPresent())
 		{
-			order.setBPartnerName(CoalesceUtil.coalesce(bPartnerLocationRecord.getBPartnerName(), userRecord.get().getName()));
-			order.setEMail(CoalesceUtil.coalesce(bPartnerLocationRecord.getEMail(), userRecord.get().getEMail()));
-			order.setPhone(CoalesceUtil.coalesce(bPartnerLocationRecord.getPhone(), userRecord.get().getPhone()));
+			final I_AD_User user = userRecord.get();
+			order.setBPartnerName(CoalesceUtil.coalesce(user.getName(), bPartnerLocationRecord.getBPartnerName()));
+			order.setEMail(CoalesceUtil.coalesce(user.getEMail(), bPartnerLocationRecord.getEMail()));
+			order.setPhone(CoalesceUtil.coalesce(user.getPhone(), bPartnerLocationRecord.getPhone()));
```

Comment:
consider using de.metas.common.util.CoalesceUtil#firstNotBlank(java.lang.String...) instead of CoalesceUtil.coalesce

Reply:
Makes sense, changed.